src/backend/distributed/sql/downgrades/citus--11.1-1--11.0-3.sql

```diff
@@ -47,3 +47,8 @@ CREATE FUNCTION pg_catalog.worker_repartition_cleanup(bigint)
 AS 'MODULE_PATHNAME', $function$worker_repartition_cleanup$function$;
 
 #include "../../../columnar/sql/downgrades/columnar--11.1-1--11.0-3.sql"
+
+DROP FUNCTION pg_catalog.get_all_active_transactions(OUT datid oid, OUT process_id int, OUT initiator_node_identifier int4,
+                                                     OUT worker_query BOOL, OUT transaction_number int8, OUT transaction_stamp timestamptz,
+                                                     OUT global_pid int8);
+#include "udfs/get_all_active_transactions/11.0-1.sql"
```

Comment:
`make install-downgrades` gives the following error:

```
fatal error: udfs/get_all_active_transactions/11.0-1.sql: No such file or directory
 #include "udfs/get_all_active_transactions/11.0-1.sql"
```

In my opinion when we fix an object during an upgrade, it is ok to leave it fixed during the downgrades. We should be fine as long as the migration scripts are idempotent.